lib/board.rb

```diff
@@ -0,0 +1,81 @@
+require 'pry'
+
+
+class Board
+
+  attr_accessor :cells, :token, :move, :player
+
+ 
+
+    def initialize
+      @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+    end
+
+
+
+    def reset!
+      @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+    end
+
+    def display
+      puts " ----------- "
+      puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+      puts " ----------- "
+      puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+      puts " -----------"
+      puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+    end   
+
+    def position(input)
+      move = input.to_i
+      move -= 1
+      @cells[move]
+    end
+
+    def full?
+      true if !@cells.detect {|x| x == " "}
```

Comment:
Use the most expressive and efficient iterator for a problem. `Board#full?` is asking if all elements are not empty strings. You can use logic that is symmetrical to those semantics with the [`#all?`](http://ruby-doc.org/core-2.2.3/Enumerable.html#method-i-all-3F) iterator.

```
cells.all? {|cell| cell != " "}`
```

This is also more efficient in that the loop will stop after the iteration for which a cell is not empty.

Finally, avoid returning boolean literals from `if` statements, relying instead on the implicit return value of the boolean expression.